packages/utils/recursive-parser.js

```diff
@@ -4,38 +4,59 @@ const utils = require("./ast-utils");
 
 module.exports = function recursiveTransform(j, ast, value, action, key) {
 	const node = utils.findRootNodesByName(j, ast, key);
+
+	// get module.exports prop
+	const root = ast
+		.find(j.ObjectExpression)
+		.filter(p => {
+			return (
+				utils.safeTraverse(p, [
+					"parentPath",
+					"value",
+					"left",
+					"object",
+					"name"
+				]) === "module" &&
+			utils.safeTraverse(p, [
+				"parentPath",
+				"value",
+				"left",
+				"property",
+				"name"
+			]) === "exports"
+			);
+		})
+		.filter(p => p.value.properties);
+
 	if (node.size() !== 0) {
-		// push to existing key
-		return ast;
+		// select node with existing key
```

Comment:
Abstract this

Reply:
Updated to use `findRootNodesByName` util method.